### PR TITLE
Release 3.5.4

### PR DESCRIPTION
**Pull request by donggyu04**
Release 3.5.4

:heavy_exclamation_mark: It must contain #778 before merging.

### CODE REVIEW

File: RELEASE-NOTE.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
+3.5.4 (2021.03.10)
+==================
+- Changes
+    - Change the HttpClient of the test script to apache http components base
```
Comment (junoyoon) [inline RELEASE-NOTE.md:4]: - Release new experimental HttpClient based on apache http5-core.

File: RELEASE-NOTE.md [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+        * Support modern HTTP specification
+        * Provide easy-to-use APIs
+    - Look up controller ip from dns when agent re-tries to connect to controller
```
Comment (junoyoon) [inline RELEASE-NOTE.md:7]: Use controller DNS instead of IP to support agent-controller reconnection in cloud env.

File: RELEASE-NOTE.md [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+    - Support dynamic modification of script template
```
Comment (junoyoon) [inline RELEASE-NOTE.md:8]: Support script template customization

File: RELEASE-NOTE.md [hunk 4]
```diff
@@ -1,3 +1,28 @@
...
+        * When creating a script, will refer to under the NGRINDER_HOME/script_template
```
Comment (junoyoon) [inline RELEASE-NOTE.md:9]: Each ngrinder admin can deploy customized script template under ${NGRINDER_HOME}/script_template

File: RELEASE-NOTE.md [hunk 5]
```diff
@@ -1,3 +1,28 @@
...
+    - Provide a test configuration to maintain the used connection on each test run
```
Comment (junoyoon) [inline RELEASE-NOTE.md:10]: Provide the connection reset option in test configuration page.
- If you turn it on, it tries to reuse connection as much as possible.

File: RELEASE-NOTE.md (hunk 1 above)
Comment (donggyu04) [reply RELEASE-NOTE.md]: @junoyoon 

I fixed all. thank you for review.

**Review by junoyoon — DISMISSED** [review]

**Review by imbyungjun — APPROVED** [review]

**Pull request merged**